prover/crates/bin/vk_setup_data_generator_server_fri/src/commitment_utils.rs

```diff
@@ -26,14 +23,6 @@ static KEYSTORE: Lazy<Mutex<Option<Keystore>>> = Lazy::new(|| Mutex::new(None));
 fn circuit_commitments(keystore: &Keystore) -> anyhow::Result<L1VerifierConfig> {
     let commitments = generate_commitments(keystore).context("generate_commitments()")?;
     Ok(L1VerifierConfig {
-        params: VerifierParams {
-            recursion_node_level_vk_hash: H256::from_str(&commitments.node)
-                .context("invalid node commitment")?,
-            recursion_leaf_level_vk_hash: H256::from_str(&commitments.leaf)
-                .context("invalid leaf commitment")?,
-            // The base layer commitment is not used in the FRI prover verification.
-            recursion_circuits_set_vks_hash: H256::zero(),
-        },
```

Comment:
I think you should calculate only `snark_wrapper` vk here and don't spend time on leaf and node here. Can be tackled separately